packages/grid/src/vaadin-grid-data-provider-mixin.js

```diff
@@ -425,12 +425,10 @@ export const DataProviderMixin = (superClass) =>
           this._debouncerApplyCachedData = Debouncer.debounce(this._debouncerApplyCachedData, timeOut.after(0), () => {
             this._setLoading(false);
 
-            iterateChildren(this.$.items, (row) => {
```

Comment:
The import for `iterateChildren` can also be removed

Reply:
Done.